### PR TITLE
Bugfix/840 superseded salt ssh

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
Fixes: #840 #843

### CODE REVIEW

File: salt/_roster/kubernetes_nodes.py [hunk 1]
```diff
@@ -20,8 +20,8 @@ def __virtual__():
         return HAS_DEPS
     else:
         return False, 'Unable to load the kubernetes nodes roster: '\
-            'dependencies kubernetes.client and/or '\
-            'kubernetes.config are unavailable.'
+            'dependencies Kubernetes Python client library '\
```
Comment (charlesprost) [inline salt/_roster/kubernetes_nodes.py]:
```suggestion
            'dependencies for Kubernetes Python client library '\
```

File: salt/_roster/kubernetes_nodes.py [hunk 2]
```diff
@@ -57,5 +57,6 @@ def targets(tgt, tgt_type='glob', **kwargs):
                 'user': annotations.get(prefix + 'user', 'root'),
                 'priv': annotations.get(prefix + 'key-path', 'salt-ssh.rsa'),
                 'sudo': bool(annotations.get(prefix + 'sudo', False)),
+                'minion_opts': {'use_superseded': ['module.run']},
```
Comment (charlesprost) [inline salt/_roster/kubernetes_nodes.py:60]: Do we need it in here if it's already a default value, see below?

File: scripts/bootstrap.sh.in [hunk 3]
```diff
@@ -163,16 +165,12 @@ sync_salt() {
         salt-run saltutil.sync_roster saltenv=metalk8s-@@VERSION
 }
 
-sync_local() {
```
Comment (charlesprost) [inline scripts/bootstrap.sh.in:166]: Was not used?

**Review by charlesprost — DISMISSED** [review]

**Review by alexandre-allard — DISMISSED** [review]

**Comment by TeddyAndrieux** [comment]
@bert-e approve

File: salt/_roster/kubernetes_nodes.py (hunk 2 above)
Comment (TeddyAndrieux) [reply salt/_roster/kubernetes_nodes.py:60]: Here, no we don't need it,
But if one day we would like to customize some others minion_opts here, that's overwrite all the value "minion_opts" so I prefer have it in both so if someone want to use another roster generated manually or if one day we add some minion_opts in the roster we have it

File: scripts/bootstrap.sh.in (hunk 3 above)
Comment (TeddyAndrieux) [reply scripts/bootstrap.sh.in:166]: Was used BUT it's already done in `configure_salt_minion_local_mode` and it's the only time where we want to sync locally

**Comment by TeddyAndrieux** [comment]
@bert-e reset

**Review by charlesprost — APPROVED** [review]

**Pull request merged**